### PR TITLE
chore: update circle-ci config

### PR DESCRIPTION
**Pull request by taylrj**

### CODE REVIEW

**Comment by babygoat** [comment]
I suggest separating the merged patches into different PR and merging first.

**Comment by taylrj** [comment]
Updated.

**Comment by taylrj** [comment]
@babygoat please see commit 118c206, which fixes problem raised by this [pipeline](https://app.circleci.com/pipelines/github/twreporter/orangutan-monorepo/236/workflows/babee021-24a7-4a4e-b421-aa8fe5b3ade1/jobs/472).

**Comment by taylrj** [comment]
Also, 13239a0 has been verified by https://github.com/twreporter/orangutan-monorepo/commit/770c9b6826bf8b7e00bf82fe78641bdaf9b2c8cb which is pushed to origin/master by `reporterdev`.

**Comment by babygoat** [comment]
> @babygoat please help to review commit [118c206](https://github.com/twreporter/twreporter-npm-packages/commit/118c206438dfe4ff8ece8351945f4b212657db24), which fixes problem raised by this [pipeline](https://app.circleci.com/pipelines/github/twreporter/orangutan-monorepo/236/workflows/babee021-24a7-4a4e-b421-aa8fe5b3ade1/jobs/472).

It seemed to me that this should be fixed manually as the publishment was not able to roll back.  What's more, I would prefer to the current behaivor as  it did not bundle the version bump operation with merge patch.

**Comment by taylrj** [comment]
Updated.

File: .circleci/config.yml [hunk 1]
```diff
@@ -71,10 +71,18 @@ commands:
               <<# parameters.prerelease >>--message "$PRERELEASE_COMMIT_HEAD"<</ parameters.prerelease >> \
               <<^ parameters.prerelease >>--message "$RELEASE_COMMIT_HEAD"<</ parameters.prerelease >> \
               --yes
-            # Push new version commit back to origin/master
-            <<^ parameters.prerelease >>git push origin master<</ parameters.prerelease >>
             echo "HEAD after version: $(git log -1 --pretty=format:%H)"
 
+  push_to_origin_master:
```
Comment (babygoat) [inline .circleci/config.yml:76]: minor. I might suggest `sync_with_remote_master` as the `origin` is just a name alias of the remote repository

File: .circleci/config.yml [hunk 2]
```diff
@@ -71,10 +71,18 @@ commands:
...
+    description: "Keep branch release synchronized with origin/master"
+    steps:
+      - run:
+          name: "Push release to origin/master"
+          command: |
+            # Synchronize branch release with origin/master as it contains the merge commit and commit 
+            # generated by lerna version
```
Comment (babygoat) [inline .circleci/config.yml:83]: Do we need to elaborate on why we need to keep the master branch up with the release branch for further version bumped by lerna?

**Comment by taylrj** [comment]
Updated.

**Comment by babygoat** [comment]
LGTM

**Pull request merged**